SatisfactorySaveEditor/ViewModel/MainViewModel.cs

```diff
@@ -523,8 +529,32 @@ private async Task LoadFile(string path)
 
         private void LoadFileAsync(string path)
         {
-            //MessageBox.Show("Loading file...");
-            saveGame = new SatisfactorySave(path);
+            try
+            {
+                saveGame = new SatisfactorySave(path);
+            }
+            catch (Exception ex)
+            {
+                if (ex is FileNotFoundException)
+                {
+                    MessageBox.Show("That file could no longer be found on the disk.\nIt has been removed from the recent files list.", "File not present", MessageBoxButton.OK, MessageBoxImage.Warning);
+                    log.Info($"Removing save file {path} from recent saves list");
+                    if (LastFiles != null && LastFiles.Contains(path))
+                    {
+                        Properties.Settings.Default.LastSaves.Remove(path);
+                        Application.Current.Dispatcher.Invoke(() =>
+                        {
+                            LastFiles.Remove(path);
+                        });
+                    }
+                }
+                else
+                {
+                    MessageBox.Show($"An error occurred while opening the file:\n{ex.Message}\n\nCheck the logs for more details.\n\nIf this issue persists, please report it via \"Help > Report an Issue\", and attach the log file and save file you were trying to open.", "Error opening file", MessageBoxButton.OK, MessageBoxImage.Error);
+                }
+                return;
```

Comment:
Check one of the log overloads (probably log.Error) that takes an exception and call it so the exception is saved to the log